packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,364 @@
+<template>
+    <div
+        ref="cookieBanner"
+        :class="[$style['c-cookieBanner'], $style['c-cookieBanner-overlay'], { [$style['c-cookieBanner--hidden']]: hideBanner }]"
+        data-cookie-consent-overlay
+        data-test-id="cookieBanner-component"
+        :aria-hidden="hideBanner">
+        <div
+            v-if="!legacyBanner"
+            :class="[$style['c-cookieBanner-card'], { [$style['c-cookieBanner-ios']]: checkIosSafari() }]">
+            <div
+                :class="$style['c-cookieBanner-content']"
+                role="dialog"
+                aria-labelledby="cookieConsentLabel"
+                aria-modal="true">
+                <h2
+                    id="cookieConsentLabel"
+                    :class="$style['c-cookieBanner-title']">
+                    <a
```

Comment:
I think this should be a button. `<button type='button'>`

Reply:
@kevinrodrigues The name anchor is only there to allow focus on the heading (<h2> is not focusable) for a11y. Can a button act as a name anchor?

Reply:
Ok. I thought the ref was focusing on mount?

Reply:
It is, but it can't focus on the `<h2>`, so I put the name anchor in - happy to look into another approach, I guess

Reply:
Link without href should trigger error for semantic/accessibility check. We shouldn't use link or button on title to make it focusable because it breaks the semantics.

Reply:
Looking at it again do we actually need it to focus on the title? I think one of the dismissable ctas makes more sense.

Reply:
@SachaZvetelman @kevinrodrigues @dandel10n Legal specifically don't want us to focus on the CTAs. If the company is seen to be pushing users to "Accept all", then that is not legal. The law states that all CTAs must have equal prominence, so if we set focus on the "Accept all", we could be seen to be forcing (disabled) users to accept all cookies. If we set focus to the "Accept only necessary" button, then we run the risk of users just hitting return and our conversion levels dropping off the cliff (as well as being open to pushing users to make a certain decision - again not legal). After discussing with legal, we agreed that the best thing to do was to focus on the title, so that all users (even screen reader users) would have the opportunity to read the cookie policy statement and have the chance to understand what the modal is about.

The only other recommendation I can think of is to have a visually-hidden link that screen readers focus to, similar to the "Skip to content" link, at the top of the cookie banner - this risks screen readers repeating the text "Cookies", as most screen reader users set the reader to read headings, unless someone can suggest some semantic text that would work for a skip link.

Happy to set up a meeting to discuss this.

Reply:
Anchor removed